docs/extensions/guides/main-extension.md

```diff
@@ -75,3 +76,25 @@ Valid values include: `"file"`, `"edit"`, `"view"`, and `"help"`.
 In this example, we simply log a message.
 However, you would typically have this navigate to a specific page or perform another operation.
 Note that pages are associated with the [`Renderer.LensExtension`](renderer-extension.md) class and can be defined in the process of extending it.
+
+The following example demonstrates how an application menu can be used to navigate to such a page:
+
+``` typescript
+import { Main } from "@k8slens/extensions";
+
+export default class SamplePageMainExtension extends Main.LensExtension {
+  appMenus = [
+    {
+      parentId: "help",
+      label: "Sample",
+      click: () => this.navigate("myGlobalPage")
+    }
+  ]
+}
+```
+
+When the menu item is clicked the `navigate()` method looks for and displays a global page with id `"myGlobalPage"`.
+This page would be defined in your extension's `Renderer.LensExtension` implmentation (See [`Renderer.LensExtension`](renderer-extension.md)).
+
+The `Main.LensExtension` class also provides the `addCatalogSource()` and `removeCatalogSource()` methods, for managing custom catalog items (or entities).
+See the [`Catalog`](tbd) documentation for full details about the catalog.
```

Comment:
That reminds me that the catalog is really two things: `CatalogCategoryRegistry` and `CatalogEntityRegistry`.

Maybe it would be a good idea to create an empty page to link to.